From d8a9e2133bb6ad518fce39a6e8a45e4440119379 Mon Sep 17 00:00:00 2001
From: shammya <kaziwasifaminshammu@gmail.com>
Date: Sat, 22 Jan 2022 11:04:45 +0600
Subject: [PATCH 1/2] branch1 commit

---
 test.cpp | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 test.cpp

diff --git a/test.cpp b/test.cpp
new file mode 100644
index 0000000..0101007
--- /dev/null
+++ b/test.cpp
@@ -0,0 +1,8 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+    cout << "first test for git patch !" << endl;
+    cout << "  branch 1 added !" << endl;
+}
\ No newline at end of file
-- 
2.25.1

